components/atom/switch/src/SwitchType/single.js

```diff
@@ -121,6 +126,10 @@ SingleSwitchTypeRender.propTypes = {
    * Is Input disabled?
    */
   disabled: PropTypes.bool,
+  /**
+   * The padding of the container is set to 0
+   */
+  isDisabledPadding: PropTypes.bool,
```

Comment:
In some components problem appears with the  padding and other components is margin. We are gonna tackle this like an epic and we might porpose a common boolean prop for all of them. My proposal is 'isFitted' (default undefined) because there is no other component which is currently using this prop and it doesn't seems to be adapted by any other for diferent purposes

fit: caber, encajar, ajustar, encajar en, adaptar, entrar